sktime/datasets/_single_problem_loaders.py

```diff
@@ -481,6 +547,12 @@ def load_basic_motions(split=None, return_X_y=True, return_type="nested_univ"):
     =BasicMotions
     """
     name = "BasicMotions"
+    if return_type == "numpy2d" or return_type == "numpyflat":
```

Comment:
numpy2d isn't one of franz scitypes I think it is 'np.ndarray' represent 2d arrays.